js/utils/index.js

```diff
@@ -75,3 +76,41 @@ export function setDeepValue(obj, path, value) {
 
   schema[pList[len - 1]] = value;
 }
+
+// http://stackoverflow.com/a/2686098/632806
+// todo: unit test
+export function abbrNum(_number, _decPlaces = 1) {
+  // 2 decimal places => 100, 3 => 1000, etc
+  const decPlaces = Math.pow(10, _decPlaces);
+  let number = _number;
+
+  // Enumerate number abbreviations
+  const abbrev = ['k', 'm', 'b', 't'];
```

Comment:
I think bringing a lib is a bit overkill just for this. Anyhow, I've just made a commit (not yet pushed) where it will build the string off of a Polyglot translation, so our translators can localize the 'k', 'm'... part and I'm using the core Intl.NumberFormat to internationalize the actual number.